community-content/prediction_featurestore_integration/prediction_featurestore_integration.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<p>try maybe</p><p>! gsutil cp -r gs://mco-mm/churn/* $BUCKET_URI</p><p> </p><p>to not need subdirectory in next line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Done.